index.js

```diff
@@ -19,11 +19,14 @@ module.exports = () => {
 		};
 	}).catch(() => {
 		return pify(getos).then(res => {
+			if (!res) {
+				res = { };
```

Comment:
If `res` is `undefined` we should throw an error with the message `Your distro couldn't be determined`, or something along those lines. And yes, if you don't want your application to crash you should do:

```
linuxDistro()
  .then(data => console.log(data))
  .catch(() => {
    // ignore error
  });
```